plugins/aep-optimize-mbox-validation/index.ts

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Feels like the name of this file doesn't match what it actually does

Reply:
@amadt Hmm basically this just makes sure that each requested mbox appears in the response. Would something like `mbox-response-events` or `mbox-response-validation` work better?

Reply:
I just didn't see anything specific to an mbox. It looks for decision scopes, but those could be an mbox or an offer. Maybe that's where I was confused.

Guess I'm not totally sure the use case. Maybe we're validation that if the user requests a certain mbox or offer, that something comes back in the response? Like if a person requests an mbox of TGT, but that mbox doesn't exist?

Reply:
@amadt Ah that makes sense, I just looked at the payloads swarna provided and thought the scopes were only associated with mboxes. Would there be a better way to validate that each requested MBox has a result? That was the task I was trying to address.